examples/stable-diffusion/training/train_text_to_image_sdxl.py

```diff
@@ -47,26 +46,25 @@
 from datasets import load_dataset
 from diffusers import (
     AutoencoderKL,
+    DDPMScheduler,
     UNet2DConditionModel,
 )
-from diffusers.loaders import LoraLoaderMixin
 from diffusers.optimization import get_scheduler
 from diffusers.training_utils import EMAModel, compute_snr
 from diffusers.utils import check_min_version, is_wandb_available
 from diffusers.utils.torch_utils import is_compiled_module
 from huggingface_hub import create_repo, upload_folder
 from packaging import version
-from peft import LoraConfig
-from peft.utils import get_peft_model_state_dict
 from torchvision import transforms
 from torchvision.transforms.functional import crop
 from tqdm.auto import tqdm
 from transformers import AutoTokenizer, PretrainedConfig
 
 from optimum.habana import GaudiConfig
 from optimum.habana.accelerate import GaudiAccelerator
+from optimum.habana.accelerate.utils.dataclasses import GaudiDistributedType
 from optimum.habana.diffusers import GaudiEulerDiscreteScheduler, GaudiStableDiffusionXLPipeline
-from optimum.habana.utils import set_seed
+from optimum.habana.utils import HabanaProfile, set_seed, to_gb_rounded
 
 
 if is_wandb_available():
```

Comment:
Let's move this in main as in https://github.com/huggingface/diffusers/blob/v0.26.3/examples/text_to_image/train_text_to_image_sdxl.py#L587.
That way we make sure `wandb` is not imported if not explicitly required by the user.

Reply:
done